<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarningPage</class>
 <widget class="QWidget" name="WarningPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panneau de Contr√¥le</string>
  </property>
  <widget class="GlobalChecker" name="generalWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="SCannerChecker" name="scannerWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="PrinterChecker" name="printerWidget" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="CNCChecker" name="cncWidget" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>701</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>721</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="touchscreen">
      <property name="text">
       <string>Calibrate Screen</string>
      </property>
      <property name="icon">
       <iconset resource="../ressources.qrc">
        <normaloff>:/img/img/touchscreen.png</normaloff>:/img/img/touchscreen.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Calibrate Plate</string>
      </property>
      <property name="icon">
       <iconset resource="../ressources.qrc">
        <normaloff>:/img/img/plate_icon.png</normaloff>:/img/img/plate_icon.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pinEditor">
      <property name="text">
       <string>Pin Editeur</string>
      </property>
      <property name="icon">
       <iconset resource="../ressources.qrc">
        <normaloff>:/img/img/pineditor.png</normaloff>:/img/img/pineditor.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="console">
      <property name="text">
       <string>Console</string>
      </property>
      <property name="icon">
       <iconset resource="../ressources.qrc">
        <normaloff>:/img/img/terminal.png</normaloff>:/img/img/terminal.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dynamicTest">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="text">
       <string>Tests</string>
      </property>
      <property name="icon">
       <iconset resource="../ressources.qrc">
        <normaloff>:/img/img/test.png</normaloff>:/img/img/test.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="configuration">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Configuration logicielle</string>
      </property>
      <property name="icon">
       <iconset resource="../ressources.qrc">
        <normaloff>:/img/img/conf.png</normaloff>:/img/img/conf.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="help">
      <property name="maximumSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="text">
       <string>Aide</string>
      </property>
      <property name="icon">
       <iconset resource="../ressources.qrc">
        <normaloff>:/img/img/help.png</normaloff>:/img/img/help.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PrinterChecker</class>
   <extends>QWidget</extends>
   <header>PrinterChecker.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CNCChecker</class>
   <extends>QWidget</extends>
   <header>CNCChecker.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SCannerChecker</class>
   <extends>QWidget</extends>
   <header>SCannerChecker.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GlobalChecker</class>
   <extends>QWidget</extends>
   <header>GlobalChecker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
